src/main/java/core/basesyntax/OperationStrategyImpl.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+
```

Comment:
Don't begin class or method implementation with empty line. Please remove the redundant empty line.